bottlecap/src/proc/clock.rs

```diff
@@ -0,0 +1,20 @@
+use std::io;
+
+#[allow(clippy::cast_sign_loss)]
+#[cfg(not(target_os = "windows"))]
+pub fn get_clk_tck() -> Result<u64, io::Error> {
+    let clk_tck = unsafe { libc::sysconf(libc::_SC_CLK_TCK) };
```

Comment:
does this change over time during an invocation? Or can we read it only once at the start?